package.json

```diff
@@ -54,7 +54,9 @@
   ],
   "dependencies": {
     "ember-cli-babel": "^5.1.6",
-    "ember-cli-sass": "5.3.1"
+    "ember-cli-htmlbars": "^1.1.0",
+    "ember-cli-sass": "5.3.1",
+    "ember-hash-helper-polyfill": "^0.1.1"
```

Comment:
we need `ember-hash-helper-polyfill` and `ember-cli-htmlbars` because of the (hash) template that passes the `hide` closure action to the yield